lib/nestful/request.rb

```diff
@@ -1,5 +1,7 @@
 module Nestful
   class Request
+    UriParser = URI.const_defined?(:Parser) ? URI::Parser.new : URI
```

Comment:
Is this a ruby 2.4 thing, or?

Reply:
It's what I saw being done already in `Nestful::Connection`